2012-07-19  Gary Benson  <gbenson@redhat.com>

	* solib-svr4.c (svr4_info): Made debug_loader_offset_p a bitfield.

diff --git a/gdb/solib-svr4.c b/gdb/solib-svr4.c
index c88b9cb..8e41f19 100644
--- a/gdb/solib-svr4.c
+++ b/gdb/solib-svr4.c
@@ -113,7 +113,7 @@ struct svr4_info
   CORE_ADDR debug_base;	/* Base of dynamic linker structures.  */
 
   /* Validity flag for debug_loader_offset.  */
-  int debug_loader_offset_p;
+  unsigned int debug_loader_offset_p : 1;
 
   /* Load address for the dynamic linker, inferred.  */
   CORE_ADDR debug_loader_offset;
